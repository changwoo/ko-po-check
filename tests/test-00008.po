# -*-text-*-
# test 8 - 조사는 띄어 쓴다
#
msgid ""
msgstr ""
"Project-Id-Version: test 8\n"
"POT-Creation-Date: 2002-08-01 01:30+0900\n"
"PO-Revision-Date: 2002-08-01 01:30+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=EUC-KR\n"
"Content-Transfer-Encoding: 8bit\n"

# E: 조사는 띄어 쓴다
#: test.c:8
#, c-format
msgid "section of `%s' conflicts with previous declaration"
msgstr "`%s' 의 섹션은 이전에 선언된 것과 충돌합니다"

# E: 조사는 띄어 쓴다
#: test.c:8
msgid "ISO C++ forbids range expressions in switch statements"
msgstr "ISO C++ 에서 switch 표현식에서의 범위식은 금지되고 있습니다"

# E: 조사는 띄어 쓴다
#: test.c:8
msgid "__builtin_saveregs not supported by this target"
msgstr "__builtin_saveregs 은 이 타겟을 지원하지 않습니다"

# E: 조사는 띄어 쓴다
#: test.c:8
msgid "function `%s' is initialized like a variable"
msgstr "함수 `%s' 이(가) 변수로 초기화됐습니다"

# 조사가 아닌 경우
#: test.c:8
msgid "I am Changwoo.  This is my wife."
msgstr "저는 창우입니다.  이 사람은 제 아내입니다."

# 조사가 아닌 경우
#: test.c:8
msgid "  -h, --help                display this help and exit\n"
msgstr "  -h, --help                이 도움말을 보여주고 끝납니다\n"
