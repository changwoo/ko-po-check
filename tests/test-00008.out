test-00008.po:19: "`%s' 의": 조사는 체언에 붙여 써야 합니다
test-00008.po:24: "C++ 에서": 조사는 체언에 붙여 써야 합니다
test-00008.po:29: "__builtin_saveregs 은": 조사는 체언에 붙여 써야 합니다
test-00008.po:34: "`%s' 이(가)": 조사는 체언에 붙여 써야 합니다
